packages/node_modules/@node-red/editor-client/src/js/ui/editor.js

```diff
@@ -2165,6 +2369,186 @@ RED.editor = (function() {
         }
     }
 
+    function arrayEq(a0, a1) {
```

Comment:
Probably quicker and easier to use `JSON.stringify(a0) === JSON.stringify(a1)` - which can be done inline rather than have another utility function that's only used in one place.

Reply:
fixed in phase3 implementation.